docs/governance/README.md

```diff
@@ -0,0 +1,116 @@
+# .NET Standard governance
+
+The .NET Standard represents the set of APIs that all .NET implementations have
+to provide. Over time, this list has to grow but it's critical that the set
+remains implementable by all current and future .NET implementations.
+
+To ensure that's the case, there is a formal process for proposing and accepting
+APIs into the .NET Standard. This document outlines the process.
+
+## Review board
+
+Today, we have several different .NET implementations:
+
+* .NET Core
+* .NET Framework
+* Mono
+* Unity
+* Xamarin
+
+Some of them have drastically different constraints, based on their runtime
+environment (for instance, Xamarin for iOS cannot perform runtime code
+generation, based on Apple's app store policy).
+
+To ensure we don't end up adding large chunks of API surface that cannot be
+implemented, we have a review board that has to sign-off on API additions to
+the .NET Standard. The board is comprised of representatives from the following
+groups:
+
+  * **.NET platform**. The rationale here is that most, if not all, of the APIs
+    that are part of .NET Standard are implemented and evolved by the .NET
+    platform team.
+  * **Xamarin & Mono**. While Mono mostly copy code from .NET Framework & .NET
+    Core base class library, changes and extensions can impact their ability to
+    support the .NET Standard. Thus, we need to coordinate any changes with
+    Mono.
+  * **Unity**. Same rationale as for Xamarin & Mono.
+
+The chairman of the review board is [@migueldeicaza](https://github.com/migueldeicaza).
+For the most part, we strive to make decisions based on consensus, but as it is
+with every group of engineers, consensus can be impossible to achieve at times
+so we need to have an ultimate tie breaker. And Miguel has a lot of expertise
+and experience building .NET implementations that are supported by multiple
+parties.
+
+Please note that the member list isn't meant to be closed: as more platform
+vendors and API drivers appear, the review board will expand accordingly.
```

Comment:
We probably should have public GitHub team "board" with the representatives. I'm not sure that having a list spelled out like in the .NET Foundation is key though. Especially because the people might change.